stablehlo/dialect/StablehloOps.td

```diff
@@ -1142,15 +1142,18 @@ def StableHLO_IfOp: StableHLO_Op<"if", [
   let summary = "If operator";
```

Comment:
Let's also delete the comment above `def StableHLO_IfOp`. It is not relevant to StableHLO.